src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -13,15 +13,207 @@
 # limitations under the License.
 """Invocation-side implementation of gRPC Asyncio Python."""
 import asyncio
-from typing import Callable, Optional
+from typing import Callable, Dict, Optional
 
+import grpc
 from grpc import _common
 from grpc._cython import cygrpc
 
 SerializingFunction = Callable[[str], bytes]
 DeserializingFunction = Callable[[bytes], str]
 
 
+class AioRpcError(grpc.RpcError):
```

Comment:
`debug_error_string` is something not listed in the API, but quite helpful for users to debug. I hope we can still propagate it to application.
See https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/_channel.py#L149